docs/sources/clients/promtail/scraping.md

```diff
@@ -373,7 +373,7 @@ scrape_configs:
 ```
 
 Only the `brokers` and `topics` is required.
-see the [configuration](../../configuration/#kafka) section for more information.
```

Comment:
We should avoid using hardcoded links.  Don't `[configuration](./configuration/#kafka)` or just `[configuration](#kafka)` work here?

Reply:
You should be able to see docs if you browse to localhost:3002/docs/loki/latest (https://github.com/grafana/loki/blob/0da555385ab87ba7a535050979950ecffb0be792/docs/Makefile#L9)

Reply:
Thanks!